packages/riipen-ui/src/components/MenuItem.jsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import clsx from "clsx";
+import css from "styled-jsx/css";
+
+import ThemeContext from "../styles/ThemeContext";
+
+import ListItem from "./ListItem";
+
+class MenuItem extends React.Component {
+  static propTypes = {
+    /**
+     * The content of the component.
+     */
+    children: PropTypes.node,
+
+    /**
+     * The function callback for when a menu item is selected
+     */
+    onClick: PropTypes.func,
+
+    /**
+     * Array or string of additional CSS classes to use.
+     *
+     * @type {string | Array}
+     */
+    classes: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
+
+    /**
+     * The type of the root object
+     */
+    type: PropTypes.string,
+
+    /**
+     * Whether or not the item is selected
+     */
+    selected: PropTypes.bool,
+
+    /**
+     * Whether or not the item is disabled
+     */
+    disabled: PropTypes.bool,
+
+    /**
+     * The color of the component. It supports those theme colors that make sense for this component.
+     */
+    color: PropTypes.oneOf(["primary", "secondary"])
+  };
+
+  static defaultProps = {
+    color: "primary"
+  };
+
+  getLinkedStyles = () => {
+    const theme = this.context;
+
+    return css.resolve`
+      .menu-item {
+        background-color: ${theme.palette.grey[100]};
+        cursor: pointer;
+      }
+      .menu-item:hover {
+        background-color: ${theme.palette.grey[200]};
+      }
+
+      .menu-item.disabled {
+        cursor: inherit;
+        background-color: ${theme.palette.grey[50]};
+        font-weight: ${theme.typography.fontWeight.light};
+      }
+
+      .menu-item.disabled:hover {
+        background-color: ${theme.palette.grey[50]};
+        color: ${theme.palette.text.secondary};
+      }
+
+      .menu-item.selected {
+        background-color: ${theme.palette.grey[200]};
+        color: ${theme.palette.text.primary};
+      }
+
+      .menu-item.selected.primary {
+        border-left: ${theme.spacing(1)}px solid ${theme.palette.primary.main};
+      }
+
+      .menu-item.selected.secondary {
+        border-left: ${theme.spacing(1)}px solid ${theme.palette.secondary.main};
+      }
+    `;
+  };
+
+  static contextType = ThemeContext;
+
+  render() {
+    const { onClick, classes, children, disabled, selected } = this.props;
+
+    const linkedStyles = this.getLinkedStyles();
+
+    const className = clsx(classes, "menu-item", linkedStyles.className, {
+      selected
+    });
+
+    const handleClick = event => {
+      if (onClick && !disabled) onClick(event);
+    };
```

Comment:
I think this is the code we can change to prevent us from having to use the `disabled` prop in `<ListItem>`. I think this function should look like:

```js
const handleClick = event => {
  if (disabled) {
    event.stopPropagation();
    event.preventDefault(); // might not be needed
  }

  if (onClick && !disabled) onClick(event);
};
```